testing/src/test/java/io/stargate/it/cql/SimpleStatementTest.java

```diff
@@ -0,0 +1,172 @@
+            .addNamedValue("k", KEY)
+            .addNamedValue("v", null)
+            .build());
+
+    Row row = session.execute("select k,v from test2 where k=?", KEY).one();
+    assertThat(row).isNotNull();
+    assertThat(row.isNull("v")).isTrue();
+  }
+
+  @Test
+  @Disabled(
+      "C* 3.11 returns an InvalidQueryException, Stargate returns a ServerError(IndexOutOfBoundsException)")
+  public void should_fail_when_named_value_missing() {
+    // For SELECT queries, all values must be filled
+    assertThatThrownBy(
+            () ->
+                session.execute(
+                    SimpleStatement.newInstance(
+                        "SELECT * from test where k = :k and v = :v", ImmutableMap.of("k", KEY))))
+        .isInstanceOf(InvalidQueryException.class);
+  }
+
+  @Test
+  public void should_use_paging_state() {
+    SimpleStatement statement = SimpleStatement.newInstance("SELECT v FROM test WHERE k=?", KEY);
+    ResultSet resultSet = session.execute(statement);
+    assertThat(resultSet.getAvailableWithoutFetching()).isEqualTo(20);
+
+    statement = statement.copy(resultSet.getExecutionInfo().getPagingState());
+    resultSet = session.execute(statement);
+
+    assertThat(resultSet.iterator().next().getInt("v")).isEqualTo(20);
+  }
+
+  @Test
+  @Disabled("C* 3.11 returns a ProtocolError, Stargate returns a ServerError")
+  public void should_throw_when_using_corrupt_paging_state() {
+    SimpleStatement statement =
+        SimpleStatement.builder("SELECT v FROM test WHERE k=?")
+            .addPositionalValue(KEY)
+            .setPagingState(ByteUtils.fromHexString("0x1234"))
+            .build();
+    assertThatThrownBy(() -> session.execute(statement)).isInstanceOf(ProtocolError.class);
+  }
+
+  @Test
+  @Disabled("Looks like the query timestamp is not propagated correctly")
+  public void should_use_query_timestamp() {
+    long timestamp = 10; // whatever
+    session.execute(
+        SimpleStatement.builder("INSERT INTO test2 (k, v) values ('test', 1)")
+            .setQueryTimestamp(timestamp)
+            .build());
+
+    Row row = session.execute("SELECT writetime(v) FROM test2 WHERE k = 'test'").one();
+    assertThat(row).isNotNull();
+    assertThat(row.getLong(0)).isEqualTo(timestamp);
+  }
+
+  @Test
+  public void should_use_tracing() {
+    SimpleStatement statement = SimpleStatement.newInstance("SELECT v FROM test WHERE k=?", KEY);
+
+    ExecutionInfo executionInfo = session.execute(statement).getExecutionInfo();
+    assertThat(executionInfo.getTracingId()).isNull();
+
+    executionInfo = session.execute(statement.setTracing(true)).getExecutionInfo();
+    assertThat(executionInfo.getTracingId()).isNotNull();
+    QueryTrace queryTrace = executionInfo.getQueryTrace();
+    assertThat(queryTrace).isNotNull();
+    assertThat(queryTrace.getEvents()).isNotEmpty();
+  }
+
+  @Test
+  public void should_use_page_size_on_statement() {
```

Comment:
Codacy found an issue: [The JUnit 5 test method name 'should_use_page_size_on_statement' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6263918)